poly_borders/help_structures.hpp

```diff
@@ -95,6 +95,9 @@ struct Polygon
   Polygon(m2::RectD const & rect, std::vector<MarkedPoint> && points)
     : m_rect(rect), m_points(std::move(points)) {}
 
+  Polygon(Polygon &&) = default;
+  Polygon & operator=(Polygon &&) = default;
```

Comment:
- Why we should declare it explicitly? 
- Put noexcept

Reply:
> Why we should declare it explicitly?

https://github.com/organicmaps/organicmaps/pull/7621#discussion_r1528583923

> Put noexcept

Looks like only `operator=` can be marked noexcept. Neither libstdc++ nor MSVC STL have `noexcept` move constructor for `std::set`:
https://github.com/gcc-mirror/gcc/blob/b313baba57f7e09f66b603e1e30dd4b48800693f/libstdc%2B%2B-v3/include/bits/stl_set.h#L233
https://github.com/microsoft/STL/blob/be81252ed1f5e5fc6d77faca0b5dbbbdae8143a2/stl/inc/set#L142